horepgd.py

```diff
@@ -69,22 +71,28 @@ def run_import(wanted_channels, tvhsocket, fetch_radio=False, nr_days=5):
         listings = Listings()
         # add listings for each of the channels
         for channel_id, channel in chmap.channel_map.items():
-            if channel['title'].lower() in (channel.lower() for channel in wanted_channels):
+            title = channel['title']
+            if title.lower() in (channel.lower() for channel in wanted_channels) or title.lower().replace(' hd', '') in (channel.lower() for channel in wanted_channels):
```

Comment:
I recommend that you do not remove any occurrence of ` hd` but only match against ' hd' being at the end of the channel name. This also applies to line 87.

Reply:
I've replaced this by checking if the channel endswith(" hd") the title is set with a substring [:-3]